packages/tests/src/api/users/oauth.ts

```diff
@@ -99,6 +104,14 @@ describe('Test oauth', function () {
       const user3 = { username: 'ROOt', password: server.store.user.password }
       await server.login.login({ user: user3, expectedStatus: HttpStatusCode.OK_200 })
     })
+
+    it('Should be able to login with an insensitive email when no similar emails exist', async function () {
```

Comment:
Can you also do an offensive test where you test you cannot login if there are multiple email (insensitive cases)